<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ripple Effect</string>
  </property>
  <widget class="GLWidget" name="glWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>251</width>
     <height>52</height>
    </rect>
   </property>
   <property name="title">
    <string>Ripple Speed</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="0" column="0">
     <widget class="QSlider" name="speedSlider">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>7</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="speedSpinBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>7</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>90</y>
     <width>251</width>
     <height>82</height>
    </rect>
   </property>
   <property name="title">
    <string>Distort Mode</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="distortVertices">
      <property name="text">
       <string>Distort Vertices</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="distortTexCoords">
      <property name="text">
       <string>Distort TextCoords</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Textures</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="imageRadioButton1">
      <property name="text">
       <string>Under Sea</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="imageRadioButton2">
      <property name="text">
       <string>Water</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="imageRadioButton3">
      <property name="text">
       <string>Wall </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>glwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
